lutris/services/amazon.py

```diff
@@ -608,48 +614,54 @@ def get_game_cmd_line(self, fuel_url):
             game_cmd, game_args = self.get_exe_and_game_args(fuel_url)
 
         if game_cmd is None:
-            game_cmd = "_xXx_AUTO_WIN32_xXx_"
+            game_cmd = AUTO_WIN32_EXE
```

Comment:
that is not going to work

Reply:
So, should I get it back as before?

Reply:
oh wait, I haven't noticed the lack of quotes.
If that's the same string in the end, then it doesn't matter.